flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -202,6 +203,12 @@ class AndroidNotificationDetails {
   /// Refer to Android notification API documentation at https://developer.android.com/reference/androidx/core/app/NotificationCompat.html#constants_2 for the available categories
   final String category;
 
+  /// Extra notification flags.
+  ///
+  /// These flags will get added to the Notification object's flags field: https://developer.android.com/reference/android/app/Notification#flags
+  /// Values are the constants that start with FLAG_: https://developer.android.com/reference/android/app/Notification#FLAG_AUTO_CANCEL
```

Comment:
For extra clarity and to improve readability, can line be changed to

`For a list of a values, refer to the documented constants prefixed with "FLAG_" (without the quotes) at https://developer.android.com/reference/android/app/Notification.html#constants_1. For example, use a value of `4` to allow the audio to repeat as documented at https://developer.android.com/reference/android/app/Notification.html#FLAG_INSISTENT`

I assume you were trying to give an example of a flag but the wording doesn't make that clear. The flag I've given in the example would be more of use as well as per the related issue. Furthermore, auto-cancel is already supported by the plugin without the use of flags

Reply:
fixed